### PR TITLE
chore: upgrade axios

### PR DESCRIPTION
**Pull request by rsioss**
Upgrade axios due to security vulnerability

### CODE REVIEW

**Comment by rsioss** [comment]
I had a failed test on my initial push, and upon further inspection it appears that the test is inherently flaky. There was a failed unit test in `soap.test.ts` due to a timestamp diff in the expected / result.

```
serializing and deserializing requests › works for types with super classes

    expect(received).toEqual(expected) // deep equality
```

Comparing the XML data, the only difference was the following: 
- Expected: `<platform_core:searchValue>2021-02-03T22:02:23.280Z</platform_core:searchValue>`
- Result: `<platform_core:searchValue>2021-02-03T22:02:23.28Z</platform_core:searchValue>`

With a non-zero last digit in the milliseconds value, the test passes. I haven't looked into how to align the formatting in the response to the expected format yet -- seems like a low-priority task. But if anyone happens to know offhand, I'd be happy to address it.

**Review by BDQ — APPROVED** [review]
Why the test change?

File: test/soap.test.ts [hunk 1]
```diff
@@ -63,7 +63,13 @@ describe("serializing and deserializing requests", () => {
...
 
   it("works for types with super classes", () => {
-    const aDate = new Date().toISOString();
+    // When a Date is marshalled to XML, and there is a trailing `0` on the ms value (such as 250),
+    // the `0` gets dropped, causing a mismatch in the expected XML vs. the result. For example:
+    //  Expected: <platform_core:searchValue>2021-02-03T22:02:23.280Z</platform_core:searchValue>
+    //  Result:   <platform_core:searchValue>2021-02-03T22:02:23.28Z</platform_core:searchValue>
+    //
+    // As a workaround, use a fixed date of: 2021-02-03T14:44:41:259Z
+    const aDate = new Date(2021, 2, 3, 14, 44, 41, 259).toISOString();
```
Comment (WojciechMatuszewskiStedi) [inline test/soap.test.ts:72]: We could leverage `fakeTimers` API exposed by `jest` (https://jestjs.io/docs/en/timer-mocks) but a simple date carries less mental overhead so, in my opinion, it is a good solution here.

**Comment by rsioss** [comment]
> Why the test change?

See my [comment above](https://github.com/Stedi/typesuite/pull/208#issuecomment-772870920) for full details -- basically it was a flaky test to begin with, that was failing intermittently depending on the time it was executed

**Review by WojciechMatuszewskiStedi — APPROVED** [review]

**Pull request merged**